### DISCUSSION TITLE
Add never expiring links when sharing a recipe

### DISCUSSION THREAD

**Discussion by Shousokon** [mealie-recipes/mealie#3055] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

It would be good to be able to share recipes via links that don't have an expiration date.

### Please provide a concise description of the feature that would resolve your issue.

In the popup to share a recipe, add a checkmark "Link never expires".

### Please consider and list out some caveats or tradeoffs made in your design decision

N/A

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Are you looking to share a single recipe with just a few people? In that case, this makes sense.

Otherwise, are you trying to just make a recipe public? If your group is set to public, and the recipe is marked as public, anyone can navigate to the recipe without a login. More info here: https://nightly.mealie.io/documentation/getting-started/usage/permissions-and-public-access/

**Reply by Shousokon** [reply]
Hey @michael-genson,

I didn't saw that this is possible. Public recipes will work fine for me 👍

Thanks

**Comment by royaltongue** [comment]
I know this is an old thread, but I'd like to have non-expiring links as well. Sometimes I want to send a single recipe to a friend, but I have no idea when they'll actually get around to looking at/making it. My current workflow is to set the expiration to something like a random date in 2070 or whatever, but this is non-ideal
